internal/controller/server/execution/job/execution_test.go

```diff
@@ -3,134 +3,29 @@ package job
 import (
 	"context"
 	"errors"
-	"octavius/internal/controller/server/repository/executor"
-	executorRepo "octavius/internal/controller/server/repository/executor"
+	"github.com/golang/protobuf/proto"
 	"octavius/internal/controller/server/repository/job"
-	"octavius/internal/controller/server/repository/metadata"
-	metadataRepo "octavius/internal/controller/server/repository/metadata"
 	"octavius/internal/controller/server/scheduler"
-	"octavius/internal/pkg/constant"
 	"octavius/internal/pkg/idgen"
 	"octavius/internal/pkg/log"
-	clientCPproto "octavius/internal/pkg/protofiles"
-	executorCPproto "octavius/internal/pkg/protofiles/executor_cp"
-	"sync"
+	"octavius/internal/pkg/protofiles"
 	"testing"
-	"time"
-
-	"github.com/golang/protobuf/proto"
 
 	"github.com/stretchr/testify/assert"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/status"
 )
 
 func init() {
 	log.Init("info", "", true, 1)
 }
 
-func TestStartExecutorHealthCheck(t *testing.T) {
-	pingChan := make(chan string)
-	sessionID := uint64(1234)
```

Comment:
Can you check if we are achieving atleast 70% coverage

Reply:
It is there in the `health` package.